src/Controllers/AddNewRouteController/index.js

```diff
@@ -0,0 +1,8 @@
+const DbService = require("../../Services/DbService");
+
+async function addNewRouteController(request, response) {
+    const collection = await DbService.connectToDb();
+    const data = await DbService.addNewRoute(collection, request.body);
```

Comment:
do we wanna check if the body has stuff in it?

Reply:
That'll be dealt with in validation